<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SearchBarWidget</class>
 <widget class="QWidget" name="SearchBarWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>461</width>
    <height>42</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SearchBarWidget</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="spacing">
    <number>12</number>
   </property>
   <item>
    <widget class="QLabel" name="labelSearchArea">
     <property name="text">
      <string>Search area:</string>
     </property>
     <property name="buddy">
      <cstring>comboBoxSearchArea</cstring>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QComboBox" name="comboBoxSearchArea"/>
   </item>
   <item>
    <widget class="QCheckBox" name="checkBoxRegEx">
     <property name="text">
      <string>Regex</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="lineEditSearch"/>
   </item>
   <item>
    <widget class="QPushButton" name="pushButtonSearch">
     <property name="text">
      <string>Search</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
